tests/pageObjects/AddPage.js

```diff
@@ -0,0 +1,41 @@
+class AddPage{
+  constructor() 
+  {
+    this.accountButtonSelector = "//button[contains(@id,'account-button')]";
+    // this.accountButtonSelector = "button#account-button";
+    this.adminPanelSelector = "//div[contains(@id,'link-to-account')]//a[contains(@href,'/admin/orders')]";
+   this.sidebarContentSelector ="a[href='#sidebar-content']";
+   this.sidebarPageSelector = "a[href='/admin/cms_pages']";
+   this.addNewPageButtonSelector = "//div[@class='page-actions pl-0 pr-0 d-none d-lg-flex']//a[@id='new_page']";
```

Comment:
Can make this selector shorter?
```suggestion
   this.addNewPageButtonSelector = "//div[@class='page-actions pl-0 pr-0 d-none d-lg-flex']//a[@id='new_page']";
```